ex2_header_css.css

```diff
@@ -0,0 +1,40 @@
+.header {
+	width: 100%;
+	height: 60px;
+	border-bottom: 1px solid;
+}
+
+.logintitle {
+	background-color: #3090C7;
+	padding: 3px;
+	text-align: center;
+	color: white;
+	box-shadow: 0px 5px 10px grey;
```

Comment:
`0` is `0`, regardless of unit of measurement. In effect, you could simply leave out the measurement unit and it would be all the same. I disagree on the statement that `vh` is relative to font size, though. Check out this [MDN article](https://developer.mozilla.org/en/docs/Web/CSS/length#Viewport-percentage_lengths) for details.